Engine/ac/dynobj/scriptoverlay.h

```diff
@@ -18,6 +18,7 @@
 
 struct ScriptOverlay final : AGSCCDynamicObject {
     int overlayId = -1;
+    int ovrIdx = 0;
```

Comment:
Default value should be -1 here, as valid storage indexes begin with 0.

Reply:
Adjusted!